readme.org

```diff
@@ -37,6 +37,7 @@ For additional detail see the [[./docs/catkit/flow/flow.org][CatFlow documentati
 * CatHUB: Catalysis Hub
 Coming soon.
 
+* CatHub: Catalysis-Hub.org interface...
```

Comment:
This looks redundant know. If you could add a brief description of the functionality here, that would be helpful.